includes/functions.php

```diff
@@ -657,6 +657,10 @@ function erp_get_date_format( $format = false ) {
  * @return string formatted date
  */
 function erp_format_date( $date, $format = false ) {
+    if ( empty( $date ) ) {
+        return;
```

Comment:
It wont be a good idea to return directly if it is being expected to get something where it is called.

Reply:
If it is called on an empty string, it returns nothing but an error. The following operation causes the error when the an empty string is provided.
```
if ( ! is_numeric( $date ) ) {
     $date = erp_current_datetime()->modify( $date )->getTimestamp();
}
```

Reply:
This function is used to convert the format of a given date. If the date is empty, what else can be expected? Given the scenario, I thought the direct return could work.

Reply:
If it is empty then you can simply return false. Cause using only return & return false make some differences in == & === comparison. And besides if you try to store this in a variable by passing empty string it will show undefined.